monkey/infection_monkey/ransomware/readme_utils.py

```diff
@@ -0,0 +1,22 @@
+import logging
```

Comment:
It's not the utility functions used by the readme feature, it is the readme feature itself. I'd rename this to `readme_dropper` or something